keyboards/ploopyco/mouse/rules.mk

```diff
@@ -1,4 +1,3 @@
-# Processor frequency
-F_CPU = 8000000
-
 POINTING_DEVICE_DRIVER = pmw3360
+
+DEFAULT_FOLDER = ploopyco/mouse/rev1_003
```

Comment:
Default folder is going away in the near future. 

This file should probably be renamed to `post_rules.mk`
```suggestion
```

Reply:
I can rename it to `post_rules.mk`. I can also rename `keyboards/ploopyco/trackball_thumb/rules.mk` to `keyboards/ploopyco/trackball_thumb/post_rules.mk`, if you think that makes sense.

Reply:
Removing the `DEFAULT-FOLDER` lines caused a bunch of lint and CI failures. I added it back in with #[f51e9f1](https://github.com/qmk/qmk_firmware/pull/24880/commits/f51e9f1f84a43876894b8fee11b49a6d9c65eb99) so that the lint and CI checks pass, but I'm open to reorganizing things if that's better.

Reply:
Yeah, if there is a rules.mk file, the build system (and CI) considers it to be a valid keyboard folder.  In this case, .... it's not.  Moving/renaming the file would probably be the best option.

Reply:
Makes sense. Committed in #[971e559](https://github.com/qmk/qmk_firmware/pull/24880/commits/971e559686e091c2cd759422bad5b87ddf0448c6).